### ISSUE TITLE
Confusing UX - Exporting project with 4+ scenes

### ISSUE THREAD

**Issue by Christopher-Hayes** [playcanvas/editor#108]
### Description

In the "Publish" > "Download New Build" the UX can be a little confusing with the download button when you have 4 or more scenes listed. When you press "Download", an HTML component is displayed below the "Download" button that the a new build is being processed + that's where you have the button that actually downloads the build. However, the panel does not scroll down when this new HTML component is displayed. If you don't know to scroll down, it appears as if the download button is broken. Took me a little while to figure out why it wasn't downloading.

Before pressing the "Download" button, the user is scrolled to the bottom of this UI panel.
![image](https://user-images.githubusercontent.com/6013871/88467899-25133780-ceaa-11ea-898d-e392fefb5124.png)

After pressing the "Download" button, notice if you look at the scrollbar the user is no longer scrolled to the bottom. If you don't notice that there's now more content below, it looks like nothing happened.
![image](https://user-images.githubusercontent.com/6013871/88467918-568c0300-ceaa-11ea-90f3-5066f3a09697.png)

After scrolling to the bottom, it now looks like this:
![image](https://user-images.githubusercontent.com/6013871/88467931-863b0b00-ceaa-11ea-86d0-8b2f02b0b364.png)

### Solutions

Automatically scroll the user to the bottom when the build starts, or even better imo, reuse the "Download" button / button container with all that horizontal space.

### Browsers

Tested on Firefox and Chrome

**Comment by willeastcott** [comment]
Yeah, this UI needs some adjustment, for sure. I've noticed this myself. Very confusing.
